clicache/src/LocalRegion.cpp

```diff
@@ -893,7 +893,7 @@ namespace Apache
 
         _GF_MG_EXCEPTION_TRY2/* due to auto replace */
 
-        apache::geode::client::VectorOfRegionEntry vc;
+			std::vector<std::shared_ptr<apache::geode::client::RegionEntry>> vc;
```

Comment:
Uhh... I guess we don't have a managed C++ linter, but can you get rid of the spaces you added here?